src/test/sql/sql_pipeline_test.cpp

```diff
@@ -445,11 +445,11 @@ TEST_F(SQLPipelineTest, UpdateWithTransactionFailure) {
   const auto sql =
       "UPDATE table_a SET a = 1 WHERE a = 12345; UPDATE table_a SET a = 1 WHERE a = 123; "
       "UPDATE table_a SET a = 1 WHERE a = 1234";
-  auto transaction_context = Hyrise::get().transaction_manager.new_transaction_context();
+  auto transaction_context = Hyrise::get().transaction_manager.new_transaction_context(false);
```

Comment:
The `false` argument is not understandable without looking into the method. Consider using an `enum class` (compare to UseMvcc::Yes/No):

https://akrzemi1.wordpress.com/2017/02/16/toggles-in-functions/